### DISCUSSION TITLE
Installation documentation has wrong paths in src/index.html

### DISCUSSION THREAD

**Discussion by AnupamKhosla** [tailwindlabs/tailwindcss#8473] [Help] [answered]
The main installation guide on https://tailwindcss.com/docs/installation suggests to use the `index.html` file in `src` folder. The `index.html` file has the following tag:   

`<link href="/dist/output.css" rel="stylesheet">`  

Since the `index.html` file reside in `src` folder, link to output.css file should be `../dist/output.css`.   

P.S: It was not obvious to me that `index.html` has to reside in `src` folder. docs should mention that in writing, not just in that black code sample box. The docs also should explain the working of npm package `tailwindcss`.   

I am under the impression now that the `dist` folder should have another `index.html` file, which I don't see being generated automatically. It would be much better to have the docs work with `index.html` in the root folder and modify `module.exports.content` accordingly.

**Comment by adamwathan** [comment, marked as answer]
Hey! The docs for that particular example are very just "give you the general idea of how this stuff works" and not meant to be real documentation for constructing a real project or where your files should go or how your web server should be configured in terms of how paths are served.

Your `index.html` file can go anywhere you want, it doesn't have to go in a `src` directory, you just have to configure everything else to make sense based on how you've structured your project.

The reason we don't put `index.html` in the root is because it can lead to slow or infinite rebuilds if people put a path like `**/*.html` in their content config, as that will now be watching everything in `node_modules` as well for example.

At the end of the day Tailwind is just a CSS framework and doesn't at all dictate how you structure your actual project, where you compile your CSS to, whether you have a `src` directory or not, etc. so I wouldn't interpret those installation instructions as this concrete unbreakable thing, they are really just a general example to help you understand how Tailwind works so you can adapt it for whatever project structure you want.

The code on the right is just an example, but the written documentation is deliberately generic and accurate:

> 1. Install tailwindcss via npm, and create your tailwind.config.js file.
> 2. Add the paths to all of your template files in your `tailwind.config.js` file.
> 3. Add the `@tailwind` directives for each of Tailwind’s layers to your main CSS file.
> 4. Run the CLI tool to scan your template files for classes and build your CSS.
> 5. Add your compiled CSS file to the `<head>` and start using Tailwind’s utility classes to style your content.

All of the paths are ultimately up to you. Use whatever templating language you want, put those templates wherever you want, compile your CSS to whatever destination you want, and configure your web server to serve those files however you want. Hope that helps!

**Comment by christianbueno1** [comment]
Well is important to know that. Thanks.
But then would be fine fix the path to `../dist/output.css`

    <link rel="stylesheet" href="../dist/output.css">
